### ISSUE TITLE
Docker installation or documentation is missing event store adapter

### ISSUE THREAD

**Issue by rommsen** [prooph/proophessor-do#65]
Today I was trying to install the docker version of proophessor-do.

It turned out that this version is not installing any adapter for the event store. so the first migration is skipped. 

Because of this registering a new user failed:
Message dispatch failed during locate-handler phase. Error: An exception was raised while creating "Prooph\ProophessorDo\Model\User\Handler\RegisterUserHandler"; no instance returned"

I had to manually install the doctrine adapter:
```
 docker run --rm -it --volume $(pwd):/app prooph/composer:7.0 composer require prooph/event-store-doctrine-adapter
```
After this everything worked fine

The version I used last week (the one still using php 5.6) worked out of the box.

If this is intended behavior the docs should be updated.

**Comment by sandrokeil** [comment]
@rommsen The intended behavior is not to deliver an adapter, because the user should choose his favourite adapter. I will update the docs.

**Comment by sandrokeil** [comment]
Fix with PR #68
